dlt/sources/helpers/rest_client/paginators.py

```diff
@@ -303,7 +321,8 @@ def __init__(
         self,
         base_page: int = 0,
         page: int = None,
-        page_param: str = "page",
+        page_param: Optional[str] = None,
+        page_body_path: Optional[jsonpath.TJsonPath] = None,
```

Comment:
Please put `page_body_path` as a last keyword argument. I overlooked this early on when there were not so many arguments, but now I plan to gradually enforce keyword-only arguments to all of paginators.